x-pack/plugins/security_solution/public/resolver/view/panels/use_format_date.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment-timezone';
+import { useUiSetting } from '../../../../../../../src/plugins/kibana_react/public';
+import { formatter, invalidDateText } from './panel_content_utilities';
```

Comment:
You could just inline these values here,  they probably shouldn't need to be in 'panel_content_utilities'

Reply:
Yea, just did that. Will be up shortly